mod_reforged/hooks/config/strings.nut

```diff
@@ -1230,10 +1229,12 @@ foreach (vanillaDesc in vanillaDescriptions)
  	}),
  	RF_Ghostlike = ::UPD.getDescription({
  		Fluff = "Blink and you\'ll miss me.",
+ 		Requirement = "Weapon with up to 4 Reach and Head and Body Armor weight up to 20"
```

Comment:
Are both parts of this effect supposed to be usable with ranged weapons?
Those weapons are considered to have 0 Reach.

E.g. I could use a loaded crossbows, move into melee range of an opponent, unload it into that enemy while profiting from the damage bonus and then move away again.

Reply:
Melee only. I'll update the description to be clear.